doc/classes/TextEdit.xml

```diff
@@ -87,6 +87,8 @@
 			<argument index="1" name="adjust_viewport" type="bool" default="true">
 			</argument>
 			<description>
+				Moves the cursor at the specified column index.
```

Comment:
"column" should be formatted as "adjust_viewport" below: surround with [code][/code].